code/datums/status_effects/buffs.dm

```diff
@@ -61,108 +54,32 @@
 /datum/status_effect/blooddrunk/on_apply()
 	. = ..()
 	if(.)
-		owner.maxHealth *= 10
-		owner.bruteloss *= 10
-		owner.fireloss *= 10
 		if(ishuman(owner))
 			var/mob/living/carbon/human/H = owner
 			for(var/X in H.bodyparts)
 				var/obj/item/organ/external/BP = X
-				BP.max_damage *= 10
-				BP.min_broken_damage *= 10
-				BP.brute_dam *= 10
-				BP.burn_dam *= 10
-		owner.toxloss *= 10
-		owner.oxyloss *= 10
-		owner.cloneloss *= 10
-		owner.staminaloss *= 10
-		owner.updatehealth()
-		last_health = owner.health
-		last_bruteloss = owner.getBruteLoss()
-		last_fireloss = owner.getFireLoss()
-		last_toxloss = owner.getToxLoss()
-		last_oxyloss = owner.getOxyLoss()
-		last_cloneloss = owner.getCloneLoss()
-		last_staminaloss = owner.getStaminaLoss()
+				BP.brute_mod *= 0.1
+				BP.burn_mod *= 0.1
```

Comment:
So the reason I didn't use the species burn and brute mods is because it'll make augments bodyparts insanely strong (and because I"m ensuring consistency with TG's behavior).

If it was *species* `brute_mod` of `*= 0.1` and you took 100 damage, it'd reduce that damage to 10---and if you had an augmented limb, it'd then reduce that damage further to 5.

Contrast this with using a *limb's* `brute_mod` of `*= 0.1`. The 100 damage will get reduced to 95 if the limb is augmented, then the further reduction reduces it down to 9.5.

The first method means you can shrug off any damage, under 50, while blood drunk, which is not the intention, at all, so I went with the second one.